tests/core_tests.py

```diff
@@ -830,12 +830,6 @@ def test_schemas_access_for_csv_upload_endpoint(
         assert data == ["this_schema_is_allowed_too"]
         self.delete_fake_db()
 
-    def test_select_star(self):
```

Comment:
Since we're keeping the `/superset/select_star` endpoint for now, let's also keep the related test. It would be good to add another test to ensure the additional database permission check is working properly on this endpoint too.